SensorMap.cpp

```diff
@@ -1,70 +1,97 @@
-/* Roborregos Maze 2020.
+/* ROBORREGOS MAZE 2020.
  * This SensorMap class has all the functions
  * To identify anything in the Map, this class
  * Works with all the sensors of the robot.
  * To get more information, go to SensorMap.h file.
 */
 #include "SensorMap.h"
 
-SensorMap::SensorMap() {
+Ultrasonic ultrasonicFrontLeft(4, 5);   // Trig, Echo.
+Ultrasonic ultrasonicFrontRight(6, 7);  // Trig, Echo.
+Ultrasonic ultrasonicRightUp(8, 9);     // Trig, Echo.
+Ultrasonic ultrasonicRightDown(10, 11); // Trig, Echo.
+Ultrasonic ultrasonicLeftUp(12, 13);    // Trig, Echo.
+Ultrasonic ultrasonicLeftDown(14, 15);  // Trig, Echo.
```

Comment:
First:
Try and use the constants as I've mentioned before:
constexpr uint8_t kTrigFrontLeft = 0;
constexpr uint8_t kEchoFrontLeft = 0;
Ultrasonic ultrasonicFronLeft(kTrigFrontLeft, kEchoFrontLeft)

Second. 
Wrap it up in an empty namespace, like:
namespace {
constexpr uint8_t kTrigFrontLeft = 0;
constexpr uint8_t kEchoFrontLeft = 0;
Ultrasonic ultrasonicFronLeft(kTrigFrontLeft, kEchoFrontLeft)
}

Reply:
If it doesn't work with the constexpr and you can't find a way to make it work, we'll go for the moment with the version you have, but still wrapped on the namespace.